SAR/template.yml

```diff
@@ -1,17 +1,19 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Transform: AWS::Serverless-2016-10-31
 Description: >
-  A deno runtime for AWS Lambda
+  Deno runtime for AWS Lambda
 
 Resources:
   DenoLambdaLayer:
-    Type: AWS::Lambda::LayerVersion
+    Type: AWS::Serverless::LayerVersion
     Properties:
       LayerName: deno
-      Description: deno-lambda DENO_LAMBDA_VERSION
-      Content:
-        S3Bucket: DENO_LAMBDA_BUCKET
-        S3Key: deno-lambda-layer_DENO_LAMBDA_VERSION.zip
+      ContentUri:
+        Bucket: DENO_LAMBDA_BUCKET
+        Key: deno-lambda-layer_DENO_LAMBDA_VERSION.zip
+      Description: deno runtime (deno-lambda@DENO_LAMBDA_VERSION)
+      LicenseInfo: MIT
+      RetentionPolicy: Retain
```

Comment:
cc @brianleroux I'm not sure about this, but the previous behavior deleted the previous layer version on every new application deploy, so I _think_ this is preferable/correct thing to do.

AWS [suggests](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-layerversion.html)

>  AWS::Lambda::LayerVersion (recommended) offers the same benefits as AWS::Serverless::LayerVersion.

but it doesn't seem possible to retain with AWS::Lambda::LayerVersion 🤷‍♂